src/engine/guidance/post_processing.cpp

```diff
@@ -199,20 +199,6 @@ bool setUpRoundabout(RouteStep &step)
     // basic entry into a roundabout
     // Special case handling, if an entry is directly tied to an exit
     const auto instruction = step.maneuver.instruction;
-    if (instruction.type == TurnType::EnterRotaryAtExit ||
-        instruction.type == TurnType::EnterRoundaboutAtExit ||
-        instruction.type == TurnType::EnterRoundaboutIntersectionAtExit)
-    {
-        step.maneuver.exit = 1;
-        // prevent futher special case handling of these two.
-        if (instruction.type == TurnType::EnterRotaryAtExit)
-            step.maneuver.instruction.type = TurnType::EnterRotary;
-        else if (instruction.type == TurnType::EnterRoundaboutAtExit)
-            step.maneuver.instruction.type = TurnType::EnterRoundabout;
-        else
-            step.maneuver.instruction.type = TurnType::EnterRoundaboutIntersection;
-    }
-
```

Comment:
this was duplicated code that seems to have gotten back in here in a broken rebase. Removed (see below for correct version)